gto-support-picasso/src/main/java/org/ccci/gto/android/common/picasso/RequestCreatorCoroutines.kt

```diff
@@ -10,14 +10,16 @@ import kotlin.coroutines.resume
 import kotlin.coroutines.resumeWithException
 import kotlinx.coroutines.CancellableContinuation
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
 import kotlinx.coroutines.suspendCancellableCoroutine
 import kotlinx.coroutines.withContext
 
 suspend fun RequestCreator.getBitmap(): Bitmap = withContext(Dispatchers.Main) {
     suspendCancellableCoroutine { cont ->
         val target = BitmapContinuationTarget(cont)
         into(target)
-        cont.invokeOnCancellation { picasso?.cancelRequest(target) }
+        cont.invokeOnCancellation { GlobalScope.launch(Dispatchers.Main.immediate) { picasso?.cancelRequest(target) } }
```

Comment:
we don't really care that this is called synchronously, it just needs to be called on the Main thread

Reply:
Does this just not work if it gets called on another thread?

Reply:
it causes a crash. https://console.firebase.google.com/u/0/project/godtools-b2f82/crashlytics/app/android:org.keynote.godtools.android/issues/87952a721eb2b0cb63bf38b4f604d897?time=last-seven-days&sessionEventKey=608B1E8401D6000111C9D19B83473054_1535133344956874408